packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts

```diff
@@ -84,6 +84,7 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
     channelService.showChannelOutput();
 
     let serverStarted = false;
+    let printedError = false;
```

Comment:
Currently we receive the same error ''EADDRINUSE: address already in use" multiple times. In addition, webruntime will kill the process with an exit code of 1. This helps prevent us from printing the same error to the user multiple times, along with the generic error from force_lightning_lwc_start_failed.

Reply:
I think we should try to handle this on the webruntime side, to make sure it only prints once and sends the proper exit code. but this looks ok for now